.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Please use yarn v1 (classic). Run `yarn set version classic` to switch versions.

Reply:
This still holds